.github/workflows/all_tests_nnpdf.yml

```diff
@@ -79,6 +82,7 @@ jobs:
         if: ${{ !cancelled() }}
         uses: actions/upload-artifact@v4
         with:
+          name: nnpdf-package-${{ github.job }}-${{ strategy.job-index }}.tar.bz2
```

Comment:
Why this and what is the purpose of keeping it as an artefact?

Reply:
You can download exactly the conda package for a given branch and use it locally.

And closes https://github.com/NNPDF/nnpdf/issues/780 :)

Reply:
So where is this uploaded, I thought env.CONDA_PREFIX would just point to the conda folder in the CI. An inline comment giving some details would be good

Reply:
the path is of the object to be uploaded, which then remains with the workflow...

but the environment is not being populated... https://github.com/NNPDF/nnpdf/actions/runs/14476915646/job/40604671855?pr=2305#step:5:15
I tested it giving it the exact path before, but within this action the environment doesn't activate it seems.

I'll fix that then merge.